cluster/manifests/ingress-controller/deployment.yaml

```diff
@@ -26,7 +26,7 @@ spec:
         operator: Exists
       containers:
       - name: controller
-        image: registry.opensource.zalan.do/teapot/kube-aws-ingress-controller:v0.3.5
+        image: registry.opensource.zalan.do/teapot/kube-ingress-aws-controller:v0.3.5
```

Comment:
Exactly @mikkeloscar. Other option is to rename the repo but then I would go straight to `alb-ingress-controller`.